docs/asciidoc/modules/ROOT/partials/usage/apoc.load.directory.adoc

```diff
@@ -0,0 +1,122 @@
+The first parameter is the pattern of file to search for.
+By default is '*', that is, search all files.
+The second is the search path of directory.
+By default is an empty string, that is, search file in `import` directory.
+
+Let's suppose we have set the config option:
+----
+apoc.import.file.use_neo4j_config=true
+----
+
+and in the `import` directory the following files:
+
+.Files
+[opts=header]
+|===
+| fileName | path
+| db.json | /
+| 1.csv | /
+| 3.csv | /
+| 4.csv | /
+| 8.xls | /
+| 11.txt |  /
+| 5.csv | subfolder1
+| 6.xls | subfolder1
+| 7.xls | subfolder1
+| 20.xls | subfolder1/foo
+| 2.csv | subfolder2
+| 9.xls | subfolder2
+| 10.txt | subfolder2
+|===
+
+We can do the following query:
+
+[source,cypher]
+----
+CALL apoc.load.directory();
+----
+
+.Results
+[opts="header"]
+|===
+| value
+| "4.csv"
+| "1.csv"
+| "db.json"
+| "3.csv"
+| "subfolder2/10.txt"
+| "subfolder2/2.csv"
+| "subfolder2/9.xls"
+| "11.txt"
+| "8.xls"
+| "subfolder1/5.csv"
+| "subfolder1/.DS_Store"
+| "subfolder1/7.xls"
+| "subfolder1/foo/20.xls"
+| "subfolder1/6.xls"
+|===
+
+We can set a file pattern, for example:
+
+[source,cypher]
+----
+CALL apoc.load.directory("*csv");
```

Comment:
Se scrivi:
`CALL apoc.load.directory("*.csv");` funziona?
In caso affermativo cambialo in questo modo per favore.